integration_test/workflow_tests/test_packages/ingest/test_no_granules.py

```diff
@@ -0,0 +1,95 @@
+import json
+import time
+import uuid
+from unittest import TestCase
+
+import boto3
+
+import helpers
+from requests import Session
+
+
+class TestNoGranules(TestCase):
+    def test_no_granules_passes(self):
+        """
+        If no granules are provided, should not store anything in DB.
+        """
+        # Set up Orca API resources
```

Comment:
possibly change lines 18-26 to helper function.

Reply:
Added helper